src/groupeddataframe/grouping.jl

```diff
@@ -87,7 +90,7 @@ function groupby(df::AbstractDataFrame, cols::Vector;
         permute!(df_groups.starts, group_perm)
         Base.permute!!(df_groups.stops, group_perm)
     end
-    GroupedDataFrame(df, cols, df_groups.rperm,
+    GroupedDataFrame(df, DataFrames.index(df)[cols], df_groups.rperm,
```

Comment:
is `DataFrames.` needed here?

Reply:
Also `index(df)` assumes that `cols` are symbols and they can be integers or vector of bools.

Reply:
AFAICT integers and Bools are accepted. Good point about the unnecessary prefix.

Reply:
Right - I do not know why it threw an error when I checked this.